temba/channels/models.py

```diff
@@ -1319,6 +1321,13 @@ def success(cls, channel, msg, msg_status, start, external_id=None, event=None,
                                       response_status=event.status_code,
                                       request_time=request_time_ms)
 
+            # Send data to Chatbase API
+            if hasattr(msg, 'is_org_connected_to_chatbase'):
+                on_transaction_commit(lambda: send_chatbase_logs.apply_async(args=(msg.chatbase_api_key,
```

Comment:
Is there a reason for doing this async? `ChannelLog.success` is going to be called from a send message task which is fairly balanced across orgs.